db/models/User.go

```diff
@@ -0,0 +1,20 @@
+package db_models
+
+import (
+	"time"
+
+	"gorm.io/gorm"
+)
+
+type User struct {
+	gorm.Model
+
+	ID       uint64 `json:"id"`
+	Name     string `json:"name"`
+	Surname  string `json:"surname"`
+	Nickname string `json:"nickname"`
+
+	CreatedAt time.Time      `json:"created_at"`
+	EditedAt  time.Time      `json:"edited_at"`
```

Comment:
Точно ли нужен это поле?

Reply:
Which field?

Reply:
`EditedAt`

Reply:
Gorm automatically creates `CreatedAt`, `EditedAt` and `DeletedAt` in database. We just specify will we use these fields in our go model. I think in User model it can bew quite useful